rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json

```diff
@@ -0,0 +1,618 @@
+                                                        "heating_source": "ELECTRIC",
+                                                        "fan": {
+                                                            "id": "Terminal Fan 1"
+                                                        },
+                                                        "primary_airflow": 471.94744319999984,
+                                                        "supply_design_heating_setpoint_temperature": 105
+                                                    }
+                                                ],
+                                                "design_thermostat_heating_setpoint": 65,
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
+                                                        "lighting_space_type": "STAIRWELL"
+                                                    }
+                                                ]
+                                            }
+                                        ],
+                                        "heating_ventilating_air_conditioning_systems": [
+                                            {
+                                                "id": "System 10",
+                                                "heating_system": {
+                                                    "id": "Heating Coil 1",
+                                                    "heating_system_type": "FURNACE",
+                                                    "energy_source_type": "ELECTRICITY"
+                                                },
+                                                "fan_system": {
+                                                    "id": "CAV Fan System 1",
+                                                    "fan_control": "CONSTANT",
+                                                    "supply_fans": [
+                                                        {
+                                                            "id": "Supply Fan 1",
+                                                            "specification_method": "SIMPLE",
+                                                            "design_electric_power": 75,
+                                                            "design_airflow": 117.98686079999996,
+                                                            "is_airflow_autosized": true
+                                                        }
+                                                    ],
+                                                    "return_fans": [
+                                                        {
+                                                            "id": "Return Fan 1"
```

Comment:
Could you add ```"specification_method": "SIMPLE",``` to all the return fans (e.g., TCD (b), (c), etc. cases)? This is the required key when the ```get_fan_system_object_supply_return_exhaust_relief_total_power_flow``` or ```get_fan_object_electric_power``` functions are used. 

Further,  please add the ```design_electric_power``` key ONLY IF it's is required for the return fans.

Reply:
@JacksonJ-KC 

I just found my comment was confusing. Could you add the ```design_electric_power``` key to all the fans (e.g., supply/return, etc.)?

Reply:
Added